tc-server/src/main/resources/com/tc/management/beans/TCServerInfoMBean.properties

```diff
@@ -1,21 +1,17 @@
-#
-#
-#  The contents of this file are subject to the Terracotta Public License Version
-#  2.0 (the "License"); You may not use this file except in compliance with the
-#  License. You may obtain a copy of the License at
-#
-#  http://terracotta.org/legal/terracotta-public-license.
-#
-#  Software distributed under the License is distributed on an "AS IS" basis,
-#  WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
-#  the specific language governing rights and limitations under the License.
-#
-#  The Covered Software is Terracotta Core.
-#
-#  The Initial Developer of the Covered Software is
-#  Terracotta, Inc., a Software AG company
-#
-#
+ # Copyright Terracotta, Inc.
```

Comment:
Does the Copyright item below need updating?

Reply:
looks correct to me.  what do you see?

Reply:
There's a line later in the file against which I cannot make a review comment 

```java
Copyright            = Internal use only; Terracotta copyright information
```

Is this one correct?

Reply:
Yes.  Correct.  That is the descriptor of the attribute.